src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -271,3 +271,132 @@ public sealed class ActivityListener : IDisposable
    }
 }
 
+namespace System.Diagnostics.Metrics
+{
+    public readonly struct Measurement<T> where T : struct
```

Comment:
Elsewhere in this file it preserves the CAS annotations.  Technically I think we can omit those since we only ship the source assembly in the package, and they aren't applied on `.NETCoreApp` where we do ship the reference assembly: https://github.com/dotnet/runtime/blob/4966c9d3f76a8056080df1764e9237bad14ecb08/src/libraries/System.Diagnostics.DiagnosticSource/pkg/System.Diagnostics.DiagnosticSource.pkgproj#L3-L6

Reply:
I'll get rid of all CAS attributes in the ref. I think this should be ok.